src/main/java/org/embulk/input/google_ads/GoogleAdsReporter.java

```diff
@@ -344,7 +344,7 @@ private Long getLoginCustomerId(String customerId)
 
     private List<Long> getLoginCustomerIds(String customerId)
     {
-        try (CustomerServiceClient client = buildClient(null).getLatestVersion().createCustomerServiceClient()) {
```

Comment:
Please revert this line. SDK 34.0.0's latest API version is v18.  So you don't need to specify specific version number.
And using getLatestVersion() is the same way as output.google_ads_* plugins.

Reply:
@yas-okadatech :
Thank you for your feedback.
I reverted it and run ETL successfully
![image](https://github.com/user-attachments/assets/a305fec8-52b8-46e4-86e2-04d2dc5eb368)

Please help me check again